frontend/src/routes/profile/+page.svelte

```diff
@@ -138,13 +154,48 @@
 </div>
 
 <h1 class="text-center text-3xl pt-5 pb-5">My providers</h1>
+<div class="flex justify-center pb-5">
+  <SearchProvidersDropdown bind:providerList={userProviders} />
+</div>
 <div class="flex justify-center">
-  {#each providers as provider}
-    <img src="{IMG_ORIGINAL}{provider.logo_path}" alt={provider.provider_name} />
-  {/each}
+  {#await getProviders() then _}
+    <div class="avatar-group -space-x-6">
+      {#each userProviders as provider}
+        <div class="static avatar">
+          <div class="sm:w-24 w-20">
+            <img
+              src="{IMG_ORIGINAL}{provider.logo_path}"
+              alt={provider.provider_name}
+            />
+          </div>
+
+          <button
+            class="absolute ml-1 btn btn-circle btn-xs btn-error z-10"
+            on:click={async () => {
+              deleteProvider(provider.provider_id)
+            }}
```

Comment:
Does this need to be a nested function?

```suggestion
            on:click={deleteProvider(provider.provider_id)}
```